playbooks/wordpress.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Deploy the OpenCraft.com server with wordpress
+  hosts: wordpress
+  become: true
+  vars:
+    - OPS_EMAIL: "ops@example.com"
```

Comment:
@Agrendalath, this is fine as long as we override it in the host-specific configuration file with the correct email address. But it might be better to remove all the variable definitions and move them to that file.

Reply:
That's right. I removed it.